ghost/admin/app/components/gh-context-menu.js

```diff
@@ -3,15 +3,67 @@ import SelectionList from '../utils/selection-list';
 import {action} from '@ember/object';
 import {inject as service} from '@ember/service';
 import {tracked} from '@glimmer/tracking';
+import {task} from 'ember-concurrency';
+
+const states = [
```

Comment:
We don't really use this list, but I thought it's useful to know what the states are 🦐

Reply:
Maybe put this in a documentation comment so that it also shows up when using this.state?

Reply:
And we could add JSDoc with a `'default'|'open'|'modal'|'loading'` value